exporters/auto/build.gradle

```diff
@@ -41,12 +42,24 @@ dependencies {
 	}
 }
 
-
 shadowJar {
+	// gradle warns since explicit or implicit dependency on this shadowJar is not declared
+	dependsOn ':shared-resourcemapping:shadowJar'
 	archiveClassifier.set('shaded')
 	mergeServiceFiles()
 }
 
+// Auto relocation for all dependencies
+tasks.named('shadowJar') {
+	// Prevent 'conscrypt' from being relocated.
+	// See https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/issues/161#issuecomment-1478831626 for details
+	// Relocating it back to its original location excludes the package from being automatically relocated to the specified prefix
+	relocate new SimpleRelocator("org.conscrypt", "org.conscrypt", new ArrayList<String>(), new ArrayList<String>())
```

Comment:
We may want to figure out WHY we depend on conscrypt and whether we can remove that transitive dependency completely....

Reply:
Does this [comment](https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/issues/161#issuecomment-1478831626) answer the 'why' here ?